pages/linux/join.md

```diff
@@ -0,0 +1,7 @@
+# join
+
+> Join lines of two sorted files on a common field.
```

Comment:
Do the files need to be sorted ? The man page says no such thing.

Reply:
Yes, they should be:
```console
$ join <(echo -e "1\n\2") <(echo -e "2\n1")
join: /dev/fd/62:2: is not sorted: 1
```